src/storage/heap_file.c

```diff
@@ -14530,23 +14531,24 @@ heap_dump_heap_file (THREAD_ENTRY * thread_p, FILE * fp, bool dump_records, cons
   status = xlocator_find_class_oid (thread_p, class_name, &class_oid, S_LOCK);
   if (status != LC_CLASSNAME_EXIST)
     {
-      return;
+      er_set (ER_WARNING_SEVERITY, ARG_FILE_LINE, ER_LC_UNKNOWN_CLASSNAME, 1, class_name);
```

Comment:
It is not an issue if a non-existent table name is entered by the user. Therefore, there seems to be no need to explicitly perform error setting (er_set()).